charts/kubernikus-system/charts/node-problem-detector/plugins/kernel-monitor.json

```diff
@@ -0,0 +1,68 @@
+{
+  "plugin": "journald",
+  "pluginConfig": {
+    "source": "kernel"
+  },
+  "logPath": "/var/log/journal",
+  "lookback": "5m",
+  "bufferSize": 10,
+  "source": "kernel-monitor",
+  "conditions": [
+    {
+      "type": "KernelDeadlock",
+      "reason": "KernelHasNoDeadlock",
+      "message": "kernel has no deadlock"
+    },
+    {
+      "type": "PodOOMKilling",
+      "reason": "NoPodOOMKilling",
+      "message": "no pods were killed after being OOM"
+    }
+  ],
+  "rules": [
+    {
+      "type": "temporary",
+      "reason": "OOMKilling",
+      "pattern": "Kill process \\d+ (.+) score \\d+ or sacrifice child\\nKilled process \\d+ (.+) total-vm:\\d+kB, anon-rss:\\d+kB, file-rss:\\d+kB"
+    },
+    {
+      "type": "temporary",
+      "condition": "PodOOMKilling",
+      "reason": "PodOOMKilling",
+      "pattern": "\\w+ cgroup out of memory: Kill process \\d+ (.+) score \\d+ or sacrifice child"
```

Comment:
the pattern we're looking for. 
another option would be something like `Memory cgroup stats for /kubepods/burstable/podfab2a136-9a7a-11e8-8f56-46884b7d303f`